app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str):
+    cd = command.split(" ")
```

Comment:
You should check if the input string contains exactly three elements and if the first element is 'cp'. This is a requirement from the checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.'